docs/core/tools/dotnet-build.md

```diff
@@ -26,9 +26,11 @@ dotnet build [-h|--help]
 
 ## Description
 
-The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries include the project's code in Intermediate Language (IL) files with a *.dll* extension and symbol files used for debugging with a *.pdb* extension. A dependencies JSON file (*.deps.json*) is produced that lists the dependencies of the application. A *.runtimeconfig.json* file is produced, which specifies the shared runtime and its version for the application.
+The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries include the project's code in Intermediate Language (IL) files with a *.dll* extension and symbol files used for debugging with a *.pdb* extension. Executable projects (as opposed to libraries) may produce additional files, such as a dependencies JSON file (*.deps.json*) that lists the dependencies of the application, and a *.runtimeconfig.json* file which specifies the shared runtime and its version for the application.
```

Comment:
With 3.0, it can also produce exe's now right? I might edit this paragraph to have a different format.